release/release_tests.yaml

```diff
@@ -4747,6 +4747,28 @@
     type: sdk_command
     file_manager: sdk
 
+#####################
+# Observability tests
+#####################
+- name: agent_stress_test
+  group: core-observability-test
+  working_dir: dashboard
+
+  stable: false
+
+  frequency: nightly
+  team: core
+  cluster:
+    cluster_env: agent_stress_app_config.yaml
+    cluster_compute: agent_stress_compute.yaml
+
+  run:
+    timeout: 14400
```

Comment:
this gonna be 4 hrs right? Is this enough?

Reply:
I think 4h is enough to see if there's bad memory leak